apps/web/src/screens/Dashboard/Solana.tsx

```diff
@@ -26,6 +27,7 @@ interface Props {
 
 export const SolanaDashboard: FC<Props> = () => {
 	const [activeTabIndex, setActiveTabIndex] = useState(0);
+	const { _id, hideBalance } = useSnapshot(settingState);
```

Comment:
because we have more than 1 place using this, will be good to make a custom hook named `useSetting` which:
- return `setting` object, including `hideBalance`
- return `updateSetting` method (created by `useCallback`)
- return other useful thing in the future

With that `useSetting` hook above, we can:
```typescript
const { setting, setSetting } = useSetting();

setting.hideBalance; // live update once something change from the db
setSetting({ hideBalance: true }); // created by `useCallback` in sync with custom-hook's dependency
```